doc/api/stream.md

```diff
@@ -868,6 +868,22 @@ If there are `'data'` listeners when `'readable'` is removed, the stream
 will start flowing, i.e. `'data'` events will be emitted without calling
 `.resume()`.
 
+##### Event: 'pause'
```

Comment:
Should be before `##### Event: 'readable'`?

Reply:
yes, will fix.